.ci.yaml

```diff
@@ -5327,6 +5327,7 @@ targets:
     recipe: devicelab/devicelab_drone
     presubmit: false
     timeout: 60
+    test_timeout_secs: "900"
```

Comment:
This needs to be under `properties` field.

Reply:
Ah right, fixed!